src/SwiftUI/SwiftUI/SwiftUIExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+using Swift;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	public unsafe static class SwiftUIExtensions
+	{
+		public static ModifiedOpacity<T> Opacity <T>(this T view, double opacity) where T : View
+		{
+			var opaqueOpacityMetadata = ModifiedOpacity<T>.SwiftType;
+			var resultPointer = Marshal.AllocHGlobal(ModifiedOpacity<T>.Size);
+			try
+			{
+				using (var viewHandle = view.GetHandle())
+				{
+					ViewOpacity(resultPointer.ToPointer(), viewHandle.Pointer, opacity, view.ViewType.Metadata, view.ViewType.GetProtocolConformance(SwiftUILib.Types.View));
+
+					return new ModifiedOpacity<T>(new TaggedPointer(resultPointer, true), opaqueOpacityMetadata);
+				}
+			}
+			catch
+			{
+				Marshal.FreeHGlobal(resultPointer);
```

Comment:
Can we stick to the mono coding conventions? I'll look into adding an .editorconfig, but for now we must do it manually...
```suggestion
			var resultPointer = Marshal.AllocHGlobal (ModifiedOpacity<T>.Size);
			try {
				using (var viewHandle = view.GetHandle ()) {
					ViewOpacity (resultPointer.ToPointer (), viewHandle.Pointer, opacity, view.ViewType.Metadata, view.ViewType.GetProtocolConformance (SwiftUILib.Types.View));
					return new ModifiedOpacity<T> (new TaggedPointer (resultPointer, true), opaqueOpacityMetadata);
				}
			} catch {
				Marshal.FreeHGlobal (resultPointer);
```

Reply:
We definitely need a `.editorconfig`